user-guide/Getting_started/DaaS/Creating_a_DaaS_system_FAQ.md

```diff
@@ -23,6 +23,9 @@ Contact [your Account Manager](https://community.dataminer.services/get-in-touch
 
 Your DaaS system along with all its associated resources and data will be irreversibly deleted.
 
+> [!NOTE]
+> If you are the owner of a DaaS system or you deployed a DaaS system, you will get an email notification 5 days before the system expires, so that you can take appropriate action to prevent losing your resources and data.
```

Comment:
It's not exactly 5 days before.
If for example one day before the expiry you use all the DataMiner credits of the organization, you will only receive the notification after that

Reply:
so maybe 'up to 5 days' or something

Reply:
How about this?

```suggestion
> If you are the owner of a DaaS system or you deployed a DaaS system, you will get an email notification if you only have enough DataMiner Credits left to keep the system running for 5 more days, so that you can take appropriate action to prevent losing your resources and data.
```

Reply:
Owner of the organization of a DaaS system or you deployed a ... *

Reply:
credits lowercase? (at least that is how it's everywhere else)

Reply:
New proposal:
```suggestion
> If you are an owner of the organization of a DaaS system or you deployed a DaaS system, you will get an email notification if your system is about to expire, for example because you do not have enough DataMiner credits left. Depending on how many DataMiner credits you use for other purposes, you will get this notification up to 5 days before the system expires. Another email notification will be sent when the system effectively expires.
```